wondrous-app/components/ProjectOnboarding/GuidesPage/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useOrgBoard } from 'utils/hooks';
+import { Container, GenericArtPanel, GenericLeftWrapper } from '../Shared';
+import { CONFIG, TYPES } from '../Shared/constants';
+import { PageLabel } from '../Shared/styles';
+import Card from './Card';
+
+const LeftPanel = () => {
+  const { setStep } = useOrgBoard();
+  const CARDS = [
+    {
+      title: 'Set up Project Basics',
```

Comment:
Just as an aside, can we set up some analytics to track the funnel for this signup flow? This way we can understand where people are dropping off in the funnel

Reply:
@Lamperoyge think this would be good to add in here.

Reply:
done, added following analytics for these steps + inner steps:

  ONBOARDING_BASICS_SETUP: 'onboarding_basics_setup',
  ONBOARDING_CORE_WORKFLOW_SETUP: 'onboarding_core_workflow_setup',
  ONBOARDING_COMMUNITY_SETUP: 'onboarding_community_setup',
  ONBOARDING_PROJECT_CREATE: 'onboarding_project_create',
  ONBOARDING_CATEGORY_SELECT: 'onboarding_category_select',
  ONBOARDING_CATEGORY_SELECT_SKIP: 'onboarding_category_select_skip',
  ONBOARDING_BIO_MEDIA_SETTINGS_SETUP: 'onboarding_bio_media_settings_setup',
  ONBOARDING_BIO_MEDIA_SETTINGS_SETUP_SKIP: 'onboarding_bio_media_settings_setup_skip',
  ONBOARDING_TWITTER_SETUP: 'onboarding_twitter_setup',
  ONBOARDING_TWITTER_SETUP_SKIP: 'onboarding_twitter_setup_skip',
  ONBOARDING_TASK_CREATE: 'onboarding_task_create',
  ONBOARDING_POD_CREATE: 'onboarding_pod_create',
  ONBOARDING_BOUNTY_CREATE: 'onboarding_bounty_create',
  ONBOARDING_MILESTONE_CREATE: 'onboarding_milestone_create',
  ONBOARDING_DISCORD_SETUP: 'onboarding_discord_setup',
  ONBOARDING_DISCORD_SETUP_SKIP: 'onboarding_discord_setup_skip',
  ONBOARDING_EMAIL_INVITES: 'onboarding_email_invites',
  ONBOARDING_INVITE_LINK_COPY: 'onboarding_invite_link_copy',
  ONBOARDING_INVITES_SKIP: 'onboarding_invites_skip',